statistics/histogram.go

```diff
@@ -1919,6 +1919,18 @@ func MergePartitionHist2GlobalHist(sc *stmtctx.StatementContext, hists []*Histog
 		globalBuckets[i].lower = globalBuckets[i-1].upper.Clone()
 		globalBuckets[i].Count = globalBuckets[i].Count + globalBuckets[i-1].Count
 	}
+
+	// Recalculate repeats
+	for _, bucket := range globalBuckets {
```

Comment:
This is not the fastest implementation. We may add comments to remind us to optimize performance later.

Reply:
A comment is added